bin/ebuild

```diff
@@ -128,14 +127,16 @@ try:
 
     if not opts.color == "y" and (
         opts.color == "n"
-        or portage.settings.get("NOCOLOR") in ("yes", "true")
+        or portage.util.no_color(portage.settings)
         or portage.settings.get("TERM") == "dumb"
         or not sys.stdout.isatty()
     ):
         portage.output.nocolor()
         portage.settings.unlock()
         portage.settings["NOCOLOR"] = "true"
```

Comment:
Do we need to set NOCOLOR here for internal use, I wonder?

Reply:
Yes I think I should remove it ... as eventually support for NOCOLOR is gonna go